src/Commons/NoReflex/Logger.hs

```diff
@@ -30,6 +33,10 @@ import Commons.NoReflex.Import
 newtype UserMessageCallback
   = UserMessageCallback { getUMsgCallback :: forall t . UserMessage t -> IO t }
 
+newtype UserSingleMsgCallback
```

Comment:
I remember we talked about this, but I forgot our decisions and conclusions. Why can't we re-use `UserMessageCallback` here? Looks like we still can recover `t ~ ()` by pattern-matching against UserMessage GADT constructors.

Reply:
erm... was it because of the `forall t` ...? not sure anymore... but I think I remember that it didn't compile..